app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @current_user_event = current_user.attended_events
+  end
```

Comment:
- [ ] The show method should separate past and upcoming events based on scope created in the event model. e.g.
```ruby
def show
    @created_events = current_user.events
    @previous_events = current_user.events.previous
    @upcoming_events = current_user.events.upcoming
end
```

Reply:
- [ ] The event with the past date should be under past events
![Screenshot (381)](https://user-images.githubusercontent.com/86371332/157934936-992cb1cf-e1ff-4d95-8719-c487eebfaf4c.png)